core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -318,7 +337,7 @@ private void buildPrimaryKey(CreateTableRequest.Builder requestBuilder, TableMet
   }
 
   private void buildSecondaryIndexes(
-      String namespace,
+      Namespace namespace,
       String table,
       CreateTableRequest.Builder requestBuilder,
       TableMetadata metadata,
```

Comment:
Since `String namespace` could either be a prefixed namespace or a nonprefixed namespace, I created the `Namespace` type to constrain it. I updated all the private methods of this class that have a namespace parameter from `String namespace` to `Namespace namespace.`